nats-server/configuration/leafnodes/README.md

```diff
@@ -1,14 +1,18 @@
 # Leaf Nodes
 
-A _Leaf Node_ extends a cluster or supercluster by bridging security domains. A leaf node proxies local messages to the cluster and cluster messages to the local server through the leaf node's connection client. The leaf node authenticates and authorizes clients using a local policy. Messages are allowed to flow to the cluster or into the leaf node based on the leaf node's connection permissions.
+A _Leaf Node_ extends a cluster or [supercluster](../gateways/README.md#gateways) by bridging security domains. A leaf node proxies local messages to the cluster and cluster messages to the local server through the leaf node's connection client. The leaf node authenticates and authorizes clients using a local policy. Messages are allowed to flow to the cluster or into the leaf node based on the leaf node's connection permissions.
```

Comment:
bridging of operator and security domains is optional. Also you can connect a leafnode to a single server.

So maybe

A LeafNode extends an existing NATS system of any size, optionally bridging both operator and security domains.

Reply:
Not a fan of the word proxy. Maybe

A leafnode server will transparently route messages as needed from local clients to the remote NATS system.

Reply:
incorporated